src/test/java/com/karankumar/bookproject/backend/entity/AuthorTest.java

```diff
@@ -50,6 +61,12 @@ public void reset(@Autowired PredefinedShelfService predefinedShelfService) {
         resetBookService();
         toRead = predefinedShelfService.findToReadShelf();
     }
+    
+  //  @Before
+  //  public void setUp() {
+    //    ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+      //  validator = factory.getValidator();
+   // }
```

Comment:
I don't think you'll need this any more, so this can be removed (and the commented out import)